### PR TITLE
Make package a module & update several dependencies to their latest versions

### PR DESCRIPTION
**Pull request by marcjansen**
This PR suggests to turn this package into pure ESM via `type: "module"` and related changes, see [this commit](https://github.com/terrestris/create-react-geo-app/pull/5/commits/24af0a7d81590a98ceb21466389ff8b95a2d24c6)

This enables us to update all dependencies to their latest versions:

    
```
@terrestris/eslint-config-typescript   ^2.1.0  →   ^3.1.0
@types/fs-extra                       ^9.0.13  →  ^11.0.1
@types/node-fetch                      ^2.6.1  →   ^2.6.2
chalk                                  ^4.1.2  →   ^5.2.0
commander                              ^9.1.0  →  ^10.0.0
eslint                                ^8.12.0  →  ^8.33.0
fs-extra                              ^10.0.1  →  ^11.1.0
node-fetch                             ^2.6.7  →   ^3.3.0
np                                     ^7.6.1  →   ^7.6.3
typescript                             ^4.6.3  →   ^4.9.5
```

Please review carefully.

### CODE REVIEW

**Comment by marcjansen** [comment]
I think this cannot be merged as is...

**Comment by marcjansen** [comment]
Once we have a CI, this will become much easier to test, I guess. Let's wait until #7 is merged.

**Review by dnlkoch — APPROVED** [review]

**Comment by marcjansen** [comment]
Thanks for the review!

**Pull request merged**
